src/mlpack/methods/ann/dists/bernoulli_distribution.hpp

```diff
@@ -0,0 +1,132 @@
+/**
+ * @file bernoulli_distribution.hpp
+ * @author Atharva Khandait
+ *
+ * Definition of the Bernoulli distribution class.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_DISTRIBUTIONS_BERNOULLI_DISTRIBUTION_HPP
+#define MLPACK_METHODS_ANN_DISTRIBUTIONS_BERNOULLI_DISTRIBUTION_HPP
+
+#include <mlpack/prereqs.hpp>
+#include "../activation_functions/logistic_function.hpp"
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * Multiple Bernoulli distributions.
```

Comment:
We should give a very small discription of Bernoulli distribution, how its defined here, for example we define the distributions by their expected value, p, given to the class by 'params' parameter. And the independent assumption between the distributions.